Doc/c-api/bool.rst

```diff
@@ -19,29 +19,30 @@ are available, however.
 
 .. c:var:: PyObject* Py_False
 
-   The Python ``False`` object.  This object has no methods.  It needs to be
-   treated just like any other object with respect to reference counts.
+   The Python ``False`` object.  This object has no methods and is immortal.
```

Comment:
Could this link to a glossary entry or docs section on immortal objects? Users may have no idea what this means.